rachellegelden/Home-Together-Canada
# HTC-621: Display Member Search Results in Profile Cards

# [HTC-621](https://github.com/rachellegelden/Home-Together-Canada/issues/621)

## Summary
- User should be able to do a search in "Search Member Profiles" and the results should be populated in the profile cards
    - Note: clicking on the profile card should not bring up the correct profile page info

## Relevant Motivation & Context
- Second last step to finish member search

## Testing Instructions
- import the following collection into postman - https://www.getpostman.com/collections/e9f690da4566ba511c37
    - see [here](https://learning.postman.com/docs/getting-started/importing-and-exporting-data/#:~:text=To%20import%20your%20API%20specifications,bring%20your%20data%20into%20Postman.) for how to do this
    - be sure to stagger the requests by setting a delay time
![image](https://user-images.githubusercontent.com/43148498/108034742-d2869d80-6fea-11eb-8777-4dbf937b8439.png)
- login to a user on the client, and select some filters (that will return users), and then click search. You should get results in the console (I'll take this console log out in the next ticket)
- the following filters should give you some results:
    ![image](https://user-images.githubusercontent.com/43148498/108138259-75cec580-7072-11eb-80ae-e9b6cf409f73.png)


## Developer checklist prior to opening this pull request:


## Reviewer
